src/components/FullscreenGrid/styles.module.scss

```diff
@@ -64,11 +64,14 @@ $expand-button-width: 40px;
   padding: $default-padding;
 }
 
+.content {
```

Comment:
This should be grouped with the existing `.content` selector:

https://github.com/mozilla/addons-code-manager/blob/2b761059604b5df35c82adea6aab59dfa9ee297c/src/components/FullscreenGrid/styles.module.scss#L62